frontends/web/src/api/account.ts

```diff
@@ -31,14 +31,18 @@ export type CoinUnit = 'BTC' | 'sat' | 'LTC' | 'ETH' | 'TBTC' | 'tsat' | 'TLTC'
 
 export type ERC20TokenUnit = 'USDT' | 'USDC' | 'LINK' | 'BAT' | 'MKR' | 'ZRX' | 'WBTC' | 'PAXG' | 'DAI';
 
+export type ERC20TokenCode = 'erc20Test' | 'eth-erc20-usdt' | 'eth-erc20-usdc' | 'eth-erc20-link' | 'eth-erc20-bat' | 'eth-erc20-mkr' | 'eth-erc20-zrx' | 'eth-erc20-wbtc' | 'eth-erc20-paxg' | 'eth-erc20-dai0x6b17';
+
+export type CoinOrTokenCode = CoinCode | ERC20TokenCode;
```

Comment:
Imo it would be more readable if we defined `CoinCode = 'btc' | ... | 'sepeth' | ERC20TokenCode`

I don't think we'll often need to differentiate between Coins and Tokens, and this would still allow to do it, right?

Reply:
I'm not 100% if I understand.

> I don't think we'll often need to differentiate between Coins and Tokens, and this would still allow to do it, right?

In add account we only allow adding Coins but not tokens.. I believe this change reflects better how the app works, and I think we sohuldn't mix tokens into CoinCodes.

Reply:
I see the point. I just don't like much having a type that is used a lot in the frontend that is called with a name like "ThisOrThat". I'd prefer to keep using the world Coin in its more general sense (btc/ltc/eth/erc20token), as I think it is more readable. What about finding a different name than 'Coin' for btc/ltc/eth and the other layer1 coins? But I can't think of a good name for that. wdyt? 

I mean smth like:
- Layer1Coin = 'btc' | 'eth' | ....
- ERC20Token = 'eth-erc20-usdt' | 'eth-erc20-usdc' | ....
- Coin = Layer1Coin | Erc20Token

But `Layer1Coin` sucks :grimacing:

Reply:
maybe `AccountCoin`?

Reply:
I agree it's not a great name, but somewhat accurate.

Accounts coin code can be a coin or token... hm. CryptoCode ? 🙃  I'll ping others maybe somebody has a nicer name.

Reply:
changed to `CoinCode = NativeCoinCode | ERC20CoinCode`

Reply:
done also squashed the commits @Beerosagos PTAL